docs/changelog.md

```diff
@@ -26,6 +26,46 @@
 .. role:: small
 ```
 
+## 2024-08-03 {small}`lamindb 0.75.1`
+
+Features.
+
+- 🚸 In Vitessce integration, enable a Vitessce button next to a dataset, also, document valid suffixes [PR](https://github.com/laminlabs/lamindb/pull/1799) [@falexwolf](https://github.com/falexwolf)
```

Comment:
I wouldn't list this here, maybe rather an update for the hub release notes once we actually have it there?